sdk/textanalytics/azure-ai-textanalytics/src/test/java/com/azure/ai/textanalytics/TextAnalyticsAsyncClientTest.java

```diff
@@ -1579,4 +1586,39 @@ public void analyzeSentimentZalgoText(HttpClient httpClient, TextAnalyticsServic
                 })).verifyComplete(), SENTIMENT_OFFSET_INPUT
         );
     }
+
+    // Healthcare LRO
+
+    @ParameterizedTest(name = DISPLAY_NAME_WITH_ARGUMENTS)
+    @MethodSource("com.azure.ai.textanalytics.TestUtils#getTestParameters")
+    public void healthcareLROWithOptions(HttpClient httpClient, TextAnalyticsServiceVersion serviceVersion) {
+        client = getTextAnalyticsAsyncClient(httpClient, serviceVersion);
+        healthcareLRO((documents, options) -> {
+            SyncPoller<TextAnalyticsOperationResult, PagedFlux<HealthcareTaskResult>>
+                syncPoller = client.beginAnalyzeHealthcare(documents, options).getSyncPoller();
+            syncPoller.waitForCompletion();
+            PagedFlux<HealthcareTaskResult> healthcareEntitiesResultCollectionPagedFlux
+                = syncPoller.getFinalResult();
+            validateHealthcareTaskResult(
+                options.isIncludeStatistics(),
+                Arrays.asList(getExpectedHealthcareTaskResult()),
+                healthcareEntitiesResultCollectionPagedFlux.toStream().collect(Collectors.toList()));
+        });
+    }
+
+    // Healthcare LRO - Cancellation
+
+    @ParameterizedTest(name = DISPLAY_NAME_WITH_ARGUMENTS)
```

Comment:
what is the purpose of the `ParameterizedTest` name? not sure how to map `DISPLAY_NAME_WITH_ARGUMENTS` with what the test is doing

Reply:
It is more for our test infra, such as RECORDING the network data.

Reply:
Thank you!